metaserver/src/main/scala/scala/meta/languageserver/ScalametaLanguageServer.scala

```diff
@@ -304,6 +318,16 @@ object ScalametaLanguageServer extends LazyLogging {
     subscriber -> semanticdbPublisher
   }
 
+  def interactiveSemanticdbStream(
+      implicit scheduler: Scheduler
+  ): (Observer.Sync[(Global, VersionedTextDocumentIdentifier, String)], Observable[semanticdb.Database]) = {
```

Comment:
I did it this way just because it was easier to retrieve the compiler in `didChange` (and consistent to the rest). We can surely introduce an intermediate data structure that is not a tuple.